tui/installer/screens.py

```diff
@@ -218,9 +218,11 @@ def get_admin_interface_configuration(answers):
 def get_installation_type(answers):
     entries = []
     for x in answers['upgradeable-products']:
-        entries.append(("Upgrade %s" % str(x), (x, x.settingsAvailable())))
```

Comment:
I think the design goal was so that you don't need to see internal device names when installing upgrading/restoring. That way it looks cleaner and doesn't show (unstable) Linux device names. You can press F5 to see the full details of the existing installation, including the disk it is installed on. Does that help?

Reply:
Right, it does give the information.  However, the goal of this patch is to help a user avoid making wrong decisions without realizing it, and I'm not sure that relying on an optional display would help much.

An example use-case where it would help, very much tied to patch series we did not submit yet, is when a system has been installed on a software RAID device (we have in XCPng a patch allowing the user to build a RAID1 for use as primary disk). I'm working on a patch-series that makes RAID assembly "opt-in, off by default" (see #16), and after this behavior change such a RAID install will not be listed at first; then if any install is found on another disk, when the disk is not mentioned in the entry itself, the user might think the install listed is the one it expects to find, and I think we don't want that to happen.

So maybe I should just move this commit back into my RAID-improvements PR-to-be (which incidentally already exists as a [PR against XCPng](https://github.com/xcp-ng/host-installer/pull/6), and on which you're welcome to comment already), with better context.

Reply:
OK, I see you have a reason for doing this. In that case I think it is generally fine to show some extra information there.

I'd like to ensure that it shows some kind of stable identifier rather than something like /dev/sda and /dev/sdb which might switch around on successive boots. Does it do this?

It might also be useful to include the disk size since that is often an easy way of disambiguating the disks in a system.

Reply:
I'm not really happy with the results on a small 80x25 textmode:
![installer-longlines](https://user-images.githubusercontent.com/1190950/207614602-0eaf1faf-7880-4a82-a3f6-99d62cb1e5a1.png)

With a longer description (faked here for testing), the screen overflow looks really bad:
![installer-verylonglines](https://user-images.githubusercontent.com/1190950/207627908-c9ab50f6-b25b-4a0d-bb7a-95949b3ef38a.png)

Unfortunately, that old newt library does not seem to support multiline list entries. Looks like for we'll have to be satisfied with just adding the size :/.